dspace-api/src/main/java/org/dspace/eperson/EPersonServiceImpl.java

```diff
@@ -113,6 +116,21 @@ public EPerson find(Context context, UUID id) throws SQLException {
         return ePersonDAO.findByID(context, EPerson.class, id);
     }
 
+    @Override
+    public EPerson findAnAdministrator(Context c)
+            throws SQLException {
+        List<EPerson> contacts = groupService.findByName(c, Group.ADMIN).getMembers();
+        EPerson currentUser;
+        if (contacts.isEmpty()) {
+            log.warn("Administrators group is empty");
+            currentUser = findByEmail(c, configurationService.getProperty("mail.admin"));
+            // Null if no such EPerson
+        } else {
+            currentUser = contacts.get(0);
```

Comment:
@mwoodiupui : Silly question perhaps, but why not just use `mail.admin` (or alternatively `alert.recipient`)?

I'm a little worried about randomly selecting the first administrator in the Admin group, as that gives no real control over where these messages go.  

So, I wonder if this whole method could be removed for simply `configurationService.getProperty("mail.admin");`  (or "alert.recipient")

But, maybe I'm misunderstanding why you aren't just using those email configs?

Reply:
Oh, I see now... You need an `EPerson` not a simple email address.  That's a bit of a pain.  I'm still finding this approach a little worrisome...there's no guarantee who the first member of the Administrator group is.